include/controller/seadControllerBase.h

```diff
@@ -20,10 +20,10 @@ class ControllerBase
     // unknown return type
     u32 getPadHoldCount(int) const;
 
-    int getButtonsTrigger() const { return mButtonsTrigger; }
-    int getButtonsRelease() const { return mButtonsRelease; }
-    int getButtonsRepeat() const { return mButtonsRepeat; }
-    int getButtonsHold() const { return mButtonsHold; }
+    const sead::BitFlag32 getButtonsTrigger() const { return mButtonsTrigger; }
```

Comment:
Const-qualifying the return type is not useful here because the value is returned by copy anyway.
```suggestion
    BitFlag32 getButtonsTrigger() const { return mButtonsTrigger; }
```
Also just BitFlag32 is enough since you're in sead.